tests/fv3_conf/ccpp_gfs_v16_flake_run.IN

```diff
@@ -0,0 +1,28 @@
+rm -fr INPUT RESTART
```

Comment:
Could this regression test setup-run-dir script be joined with an existing script, using variables to control copying the ORO_FLAKE data?

Reply:
I'd prefer to have this new feature separate from gfs_v16 as the feature does not belong to gfs_v16, but based on gfs_v16. In the future it can be one of candidate to be included in next production branch, srw, RRFS, or gfsv17.

Reply:
Ok, all good.